main.tf

```diff
@@ -66,10 +66,6 @@ resource "google_redis_instance" "default" {
       rdb_snapshot_period = persistence_config.value["rdb_snapshot_period"]
     }
   }
-
-  lifecycle {
-    ignore_changes = [maintenance_schedule]
```

Comment:
Should it just be ` [maintenance_policy]` ??

Reply:
@dubb-b not sure I follow. It's currently set to `maintenance_policy`?

https://github.com/terraform-google-modules/terraform-google-memorystore/blob/531de8fe9218900117be02117982926ae50f25bb/main.tf#L70-L72

Reply:
Oh I see what you mean. This PR is about reverting the lifecycle policy as `maintenance_schedule` is now correctly marked as output only (fixed in https://github.com/GoogleCloudPlatform/magic-modules/pull/8261).